pandas/tests/arrays/string_/test_string.py

```diff
@@ -314,6 +314,22 @@ def test_constructor_raises(cls):
         cls(np.array(["a", pd.NaT], dtype=object))
 
 
+@td.skip_if_no("pyarrow", min_version="1.0.0")
+def test_constructor_pyarrow_not_string_raises(cls):
```

Comment:
Maybe this test can be ArrowStringArray specific? 
Do we have a separate test file here for some tests that are speficic to it? (I think you can add a `tests/arrays/string_/test_string_arrow.py`

Reply:
added